### PR TITLE
jormungandr: add traveller profiles

### PR DESCRIPTION
**Pull request by kinnou02**

### CODE REVIEW

File: source/jormungandr/jormungandr/travellers_profile.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    'fast_walker': TravellerProfile(walking_speed=1.67, bike_speed=5, max_duration_to_pt=20*60, first_and_last_section_mode=['walking', 'bike', 'bss']),
+    'stroller': TravellerProfile(walking_speed=1.11, max_duration_to_pt=15*60, first_and_last_section_mode=['walking']),
+    'wheelchair': TravellerProfile(walking_speed=0.83, max_duration_to_pt=20*60, first_and_last_section_mode=['walking'], wheelchair=True),
+    'luggage': TravellerProfile(walking_speed=1.11, max_duration_to_pt=15*60, first_and_last_section_mode=['walking']),
+    'heels': TravellerProfile(walking_speed=1.11, bike_speed=4.17, max_duration_to_pt=15*60, first_and_last_section_mode=['walking', 'bike', 'bss']),
+    'scooter': TravellerProfile(walking_speed=2.22, max_duration_to_pt=15*60, first_and_last_section_mode=['walking']),
+    'cyclist': TravellerProfile(walking_speed=1.39, bike_speed=4.17, max_duration_to_pt=12*60, first_and_last_section_mode=['walking', 'bike', 'bss']),
+    'motorist': TravellerProfile(walking_speed=1.11, car_speed=11.11, max_duration_to_pt=15*60, first_and_last_section_mode=['walking', 'car']),
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/travellers_profile.py]: the car should be only for the departure

File: source/jormungandr/jormungandr/interfaces/v1/Journeys.py [hunk 2]
```diff
@@ -579,6 +596,9 @@ def __init__(self):
     @ManageError()
     def get(self, region=None, lon=None, lat=None, uri=None):
         args = self.parsers['get'].parse_args()
+
+        override_params_from_traveller_type(args)
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/interfaces/v1/Journeys.py]: you should implement it also for the post

File: source/jormungandr/jormungandr/interfaces/v1/Journeys.py [hunk 3]
```diff
@@ -485,6 +486,21 @@ def compute_regions(args):
 
     return regions
 
+def override_params_from_traveller_type(args):
```
Comment (stifoon) [inline source/jormungandr/jormungandr/interfaces/v1/Journeys.py]: could you change with "traveler_type" (1 "L")

File: source/jormungandr/jormungandr/travellers_profile.py [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+    @property
+    def bss_speed(self):
+        return self.bike_speed
+
+travellers_profile = {
+    'standard': TravellerProfile(walking_speed=1.39,
+                                 bike_speed=4.17,
+                                 max_duration_to_pt=12*60,
+                                 first_and_last_section_mode=['walking', 'bike', 'bss']),
```
Comment (stifoon) [inline source/jormungandr/jormungandr/travellers_profile.py]: only ['walking', 'bss'] on that profile

File: source/jormungandr/jormungandr/travellers_profile.py [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+    'slow_walker': TravellerProfile(walking_speed=0.83,
+                                    bike_speed=2.78,
+                                    max_duration_to_pt=20*60,
+                                    first_and_last_section_mode=['walking', 'bike', 'bss']),
+
+    'fast_walker': TravellerProfile(walking_speed=1.67,
+                                    bike_speed=5,
+                                    max_duration_to_pt=20*60,
+                                    first_and_last_section_mode=['walking', 'bike', 'bss']),
```
Comment (stifoon) [inline source/jormungandr/jormungandr/travellers_profile.py]: only ['walking', 'bss'] on that profile

File: source/jormungandr/jormungandr/travellers_profile.py [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+travellers_profile = {
+    'standard': TravellerProfile(walking_speed=1.39,
+                                 bike_speed=4.17,
+                                 max_duration_to_pt=12*60,
+                                 first_and_last_section_mode=['walking', 'bike', 'bss']),
+
+    'slow_walker': TravellerProfile(walking_speed=0.83,
+                                    bike_speed=2.78,
+                                    max_duration_to_pt=20*60,
+                                    first_and_last_section_mode=['walking', 'bike', 'bss']),
```
Comment (stifoon) [inline source/jormungandr/jormungandr/travellers_profile.py]: only ['walking'] please

File: source/jormungandr/jormungandr/travellers_profile.py [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+                                   wheelchair=True),
+
+    'luggage': TravellerProfile(walking_speed=1.11,
+                                max_duration_to_pt=15*60,
+                                first_and_last_section_mode=['walking']),
+
+    'heels': TravellerProfile(walking_speed=1.11,
+                              bike_speed=4.17,
+                              max_duration_to_pt=15*60,
+                              first_and_last_section_mode=['walking', 'bike', 'bss']),
```
Comment (stifoon) [inline source/jormungandr/jormungandr/travellers_profile.py]: only ['walking', 'bss'] please

File: source/jormungandr/jormungandr/travellers_profile.py [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+
+    'scooter': TravellerProfile(walking_speed=2.22,
+                                max_duration_to_pt=15*60,
+                                first_and_last_section_mode=['walking']),
+
+    'cyclist': TravellerProfile(walking_speed=1.39,
+                                bike_speed=4.17,
+                                max_duration_to_pt=12*60,
+                                first_and_last_section_mode=['walking', 'bike', 'bss']),
```
Comment (stifoon) [inline source/jormungandr/jormungandr/travellers_profile.py]: no ['bike'] on last_section_mode please

**Pull request merged**